code/lgb.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel='icon' href='images/favicon.ico' type='image/ico'/ >
+    <title>Website LGB - Home</title>
+    <link rel="stylesheet" type="text/css" href="./style.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="navigation-logo">
+      <a href="index.html" class="link-image"> <img class="image-logo" src="images/LGB-logo.png"></a>
+    </div>
+    <div class="navigation">
+      <a href="lgb.html" class="active">Über den LGB</a>
+      <a href="mitglieder.html">Mitglieder</a>
+      <a href="veranstaltungen.html">Veranstaltungen</a>
+      <a href="impressionen.html">Impressionen</a>
+      <a href="kontakt.html">Kontakt</a>
+    </div>    
+  <div>
+      <div class="div-content-image">
+        <img class="image-content" src="images/group-picture.jpg" alt="Gruppenbild mit Mitgliedern des LGB">
+      </div>
+      <div class="div-content-left"> 
+        <h1 class="h1-content">Die Gönnervereinigung LGB</h1>
+          <h2>Seit 1991 unterstützen die Mitglieder des LGB (Leichtathletik-Gönnerverein des LC Brühl) die Aktivitäten des LC Brühl Leichtathletik mit jährlichen finanziellen Zuschüssen. </h2>
+          <p>Von diesen namhaften finanziellen Leistungen profitieren sowohl die Nachwuchs- und Leistungssportabteilungen als auch einzelne Spitzenathleten / Spitzenathletinnen des LC Brühl. Dank diesen Finanzspritzen wurden in der Vergangenheit viele Projekte realisiert und werden in Zukunft auch weitere und erfolgsversprechende ermöglicht.</p>
+          <h3>Gemütliches Beisammensein im Fokus</h3>  
+          <p>Die eigentlichen Vereinsaktivitäten bilden aber die einzelnen Veranstaltungen, an denen sich die LGB-Mitglieder unter dem Motto «Lachen und gemütliches Beisammensein» treffen. </p>
+      </div>
+    </div>
+    <div>
+      <div class="div-content-right"> 
+          <h3>Der Vorstand des LGB setzt sich aus den folgenden Personen zusammen:</h>
+            <table>
+            <tbody>
+              <tr>
+                <td class="td-image"><img class="image-portrait" src="images/markus-wirth.jpg" alt="Portrait von Markus Wirth, Präsident">
```

Comment:
This table looks great! 

If you want the images to be a little smaller, you could add a max-width, or set the width to a percentage. 🙂 If you add a class to your css-file and add it to all the images, you could set a width to all pictures at once.